inference-engine/src/cldnn_engine/ops/parameter.cpp

```diff
@@ -0,0 +1,260 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "cldnn_program.h"
+#include "cldnn_common_utils.h"
+
+#include "ngraph/op/parameter.hpp"
+
+#include "api/input_layout.hpp"
+#include "api/reorder.hpp"
+#include "api/data.hpp"
+
+using namespace InferenceEngine;
+
+namespace CLDNNPlugin {
+
+void CreateParameterOp(Program& p, const std::shared_ptr<ngraph::Node>& node) {
+    auto op = std::dynamic_pointer_cast<ngraph::op::v0::Parameter>(node);
+    if (!op)
+        THROW_IE_EXCEPTION << INVALID_OP_MESSAGE;
+
+    auto networkInputs = p.GetNetworkInputs();
+    if (networkInputs.find(op->get_friendly_name()) == networkInputs.end()) {
+        THROW_IE_EXCEPTION << "Can't find input " << op->get_friendly_name() << " in InputsDataMap";
+    }
+    auto inputInfo = networkInputs.at(op->get_friendly_name());
+     // first create and add the input layout
+    const auto inputDesc = inputInfo->getTensorDesc();
+    const auto inputDims = inputDesc.getDims();
+    Layout l = inputDesc.getLayout();
+    Precision ip = inputDesc.getPrecision();
+
+    cldnn::format inputFormat = cldnn::format::bfyx;
+    if (InferenceEngine::Layout::BLOCKED == l && 6 == inputDims.size()) {
+        inputFormat = cldnn::format::bfwzyx;
```

Comment:
I'm not sure that we need another using directives here. 
Since we have many similar structures for the same things defined in different components, explicit namespace resolution might help to understand what is really happening in the code. 
E.g.
```cpp
static cldnn::nearest_mode GetNearestMode(ngraph::op::v4::Interpolate::NearestMode mode) {
    switch (mode) {
    case ngraph::op::v4::Interpolate::NearestMode::round_prefer_floor:
        return cldnn::nearest_mode::round_prefer_floor;
...
```

and 
```cpp
using namespace ngraph;
using namespace cldnn;

static nearest_mode GetNearestMode(op::v4::Interpolate::NearestMode mode) {
    switch (mode) {
    case op::v4::Interpolate::NearestMode::round_prefer_floor:
        return nearest_mode::round_prefer_floor;
...
```

Second version is a bit shorter indeed, but I can't say that the readability becomes better in that case. Also, now we have to remember that `nearest_mode` is defined somewhere in cldnn and we do the conversion of ngraph parameter into cldnn one. Maybe it's obvious for us (what else we can do in the plugin), but for someone not familiar with the plugin explicit namespace might be useful.